pacemaker_metrics.go

```diff
@@ -211,3 +220,24 @@ func (c *pacemakerCollector) recordResourcesMetrics(node node, ch chan<- prometh
 		}
 	}
 }
+
+func (c *pacemakerCollector) recordFailCountMetrics(pacemakerStatus pacemakerStatus, ch chan<- prometheus.Metric) {
+	for _, node := range pacemakerStatus.NodeHistory.Node {
+		for _, resHistory := range node.ResourceHistory {
+			// if value is 1000000 this is a special value in pacemaker which is infinity fail count
+			if resHistory.FailCount == 1000000 {
+				ch <- c.makeGaugeMetric("fail_count", math.Inf(resHistory.FailCount), node.Name, resHistory.Name)
+				continue
+			}
+			ch <- c.makeGaugeMetric("fail_count", float64(resHistory.FailCount), node.Name, resHistory.Name)
```

Comment:
I think this could be more readable if we extracted the value of the metric in a variable and assigned it in the body of the if block, rather than repeating the construction of the metric.

Reply:
Yo I have thought on this  actually. I'm ok with the var

Reply:
done